core/new-gui/src/app/dashboard/user/type/workflow-executions-entry.ts

```diff
@@ -9,4 +9,5 @@ export interface WorkflowExecutionsEntry {
   status: number;
   result: string;
   bookmarked: boolean;
+  logLocation: string;
```

Comment:
Why do we need this information on frontend? Can we do the check on backend?

Reply:
this information is necessary for the frontend to identify which execution can be replayed.